api/nakadi-event-bus-api.yaml

```diff
@@ -1260,3 +1272,51 @@ definitions:
         description: Documentation for the enrichment.
     required:
       - name
+
+
+  BatchItemResponse:
+    description: |
+      A status corresponding to one individual Event's publishing attempt.
+    properties:
+      eid:
+        type: string
+        format: uuid
+        description: |
+          eid of the corresponding item.
+      publishing_status:
+        type: string
+        enum:
+          - FAILED
+          - SUBMITTED
+          - ABORTED
+        description: |
+          Indicator of the submission of the Event within a Batch.
+          - SUBMITTED indicates successful submission, including commit on he underlying broker.
+          - FAILED indicates the message submission was not possible and can be resubmitted if so desired.
+          - ABORTED indicates that no further attempt at publishing the affected item was attempted because
+            of a failure on another item.
+      step:
+        type: string
+        enum:
+          - NONE
+          - VALIDATION
+          - ENRICHMENT
+          - PARTITIONING
+          - PUBLISHING
+        description: |
+          Indicator of the last step in the pulbishing process this Event reached.
```

Comment:
Inclusive or exclusive? For example, if it's inclusive, it means that an item with step VALIDATION had a validation issue. If it's exclusive, then it wasn't even validated.

From what I understand, this is an "inclusive" semantics. Wdyt about making it even more obvious "this Event reached, inclusively"?